docs/sources/awscloudwatch.md

```diff
@@ -44,9 +44,9 @@ spec:
 
   sink:
     ref:
-      apiVersion: eventing.knative.dev/v1
-      kind: Broker
-      name: default
+      apiVersion: eventing.triggermesh.io/v1alpha1
+      kind: RedisBroker
```

Comment:
I have not seen RedisBroker referenced or example implementations anywhere else in the guides for the docs (maybe missing it) but I think it would be good to move this to the MemoryBroker for both consistency and easy of setup/use for readers following along.